Libs/MRML/Core/vtkMRMLScalarVolumeDisplayNode.cxx

```diff
@@ -372,8 +373,6 @@ void vtkMRMLScalarVolumeDisplayNode::Copy(vtkMRMLNode *anode)
     this->AddWindowLevelPreset(node->GetWindowPreset(p), node->GetLevelPreset(p));
     }
 
-  this->EndModify(disabledModify);
```

Comment:
I haven’t checked window level presets yet.

Are you saying those above lines should be removed from CopyContent because we don’t want the copied display nodes to have all the window/level presets as well?  Or to keep it as long as the performance is good by some metric?

Reply:
The presets should be copied, that's good. The code just looked suspicious (I did not see removal of old presets).